### DISCUSSION TITLE
Cannot find preset's package

### DISCUSSION THREAD

**Discussion by tcurdt** [renovatebot/renovate#26229] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

For this repo

https://github.com/tcurdt/nixcfg/blob/main/renovate.json

the bot is telling me

> There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.
>
> Error type: Cannot find preset's package (github>tcurdt/renovate-config)

but the package is public and available

https://github.com/tcurdt/renovate-config/blob/main/default.json

It seems to be conforming [to the docs](https://docs.renovatebot.com/config-presets/#how-to-use-preset-configs).

What am I missing? The error message isn't particular specific.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
It is not public and available

**Reply by tcurdt** [reply]
Argh. Sorry, only checked the other one. It's public now.

**Reply by tcurdt** [reply]
...but that should have not affected the renovatebot, right?

**Reply by rarkins** [reply]
A public repo cannot access a private preset in the same org

**Reply by tcurdt** [reply]
That's a little odd - after all the bot has access to both. But fair enough.
Let's see if it works now.
